components/ui/Footer.js

```diff
@@ -0,0 +1,87 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import Link from 'next/link'
+import { StyledLinkText, Body1 } from './Typography'
+import theme from '../../util/theme'
+
+const FooterContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 8rem;
+
+  background: ${props => props.backgroundColor}
+    ${props => props.theme.halftones.md};
+`
+
+const FooterLinksContainer = styled.div`
+  display: flex;
+  font-size: 2rem;
+  line-height: 2rem;
+
+  > :not([hidden]) ~ :not([hidden]) {
+    border-right-width: 0;
+    border-left-width: 0.1rem;
+    border-left-style: solid;
+    border-color: ${theme.colors.white};
+  }
+`
+
+const FooterLink = styled.div`
+  padding: 0 1.2rem;
+`
+
+const Copyright = styled.div`
+  padding-top: 1.6rem;
+  color: ${theme.colors.white};
+`
+
+const Footer = ({ backgroundColor }) => (
+  <FooterContainer backgroundColor={backgroundColor}>
+    <FooterLinksContainer>
+      <FooterLink>
+        <Link href="#TODO" passHref>
```

Comment:
I don't think we know the final URL, correct?
Also, is it weird to link to the same website from ... the website?
Or is this some other website?

Reply:
OK... this prompted a quick discussion... new links (updated in the Figma too):

1. Whitepaper
2. Twitter (https://twitter.com/lvlprotocol)
3. IndieDAO (https://twitter.com/theindiedao)
4. Gitbook (https://indiedao.gitbook.io/indiedao/)